### PR TITLE
TRD update raw stats and tracklets task fix

### PR DESCRIPTION
**Pull request by martenole**
Can be merged only after https://github.com/AliceO2Group/AliceO2/pull/11754 is merged in O2.

### CODE REVIEW

File: Modules/TRD/src/TrackletsTask.cxx [hunk 1]
```diff
@@ -244,8 +244,19 @@ void TrackletsTask::buildHistograms()
...
+  float xBinLogMax = 8.f;
+  float logBinWidth = (xBinValMax - xBinValMin) / nLogBins;
+  for (int iBin = 0; iBin <= nLogBins; ++iBin) {
+    xBins[iBin] = TMath::Power(10, xBinValMin + iBin * logBinWidth);
+  }
+  mTrackletsPerTimeFrame = new TH1F("trackletspertimeframe", "Number of Tracklets per timeframe;Tracklets in TimeFrame;Counts", nLogBins, xBins);
+  mTrackletsPerTimeFrame->SetStats(11111);
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrame);
+  getObjectsManager()->setDefaultDrawOptions(mTrackletsPerTimeFrame->GetName(), "logx");
+  // getObjectsManager()->setDisplayHint(mTrackletsPerTimeFrame->GetName(), "SetOptStat(11111)"); // FIXME: how to show number of underflow bin entries by default?
```
Comment (martenole) [inline Modules/TRD/src/TrackletsTask.cxx:259]: @knopers8 is it possible to show the underflow bin count be default in QCG?

Comment (knopers8) [reply Modules/TRD/src/TrackletsTask.cxx:259]: Hm, I don't know myself. The display hint and default draw options are passed by QCG to JSROOT. Is there something else needed to allow for it?

Comment (martenole) [reply Modules/TRD/src/TrackletsTask.cxx:259]: Ok, no problem. I'll try to play around a bit and see if I can make it work in the QCG test.

**Comment by martenole** [comment]
Could this be merged please?

**Comment by martenole** [comment]
Ping @knopers8 and @Barthelemy

**Pull request merged**